static/templates/settings/linkifier_settings_admin.hbs

```diff
@@ -77,5 +77,6 @@
                 <tbody id="admin_linkifiers_table" class="required-text" data-empty="{{t 'No linkifiers set.' }}"></tbody>
             </table>
         </div>
+        <div id="admin_page_linkifiers_loading_indicator"></div>
```

Comment:
Since we don't hit the server to get linkifiers, and since there are never gonna be lots to render, I think we should just kill off the loading indicator.

Reply:
Makes sense, yes. Done, thanks!
(Although we could have a good number of linkifiers once we have stream-specific ones? 🤔 But even then it should be pretty fast IG.)

Reply:
I think anything below thousands will render faster than perception, and I'm not convinced thousands would actually work.